Dialog docs (#2409)
microsoft/teams-ai: book/src/5.in-depth-guides/2.dialogs/2.handling-dialog-submissions.md

# Handling Dialog Submissions

> Dialogs have a specific `dialog.submit` event to handle submissions. When a user submits a form inside a dialog, the app is notified via this event. Your application handles this event to process the submission values, and can decide to send a response back or proceed to more steps in the dialogs (see [Multi-step Dialogs](./3.handling-multi-step-forms.md)).

Reviewer: ```suggestion
Dialogs have a specific `dialog.submit` event to handle submissions. When a user submits a form inside a dialog, the app is notified via this event, which is then handled to process the submission values, and can either send a response or proceed to more steps in the dialogs (see [Multi-step Dialogs](./3.handling-multi-step-forms.md)).
```

Author: ty for editing!
